src/models/OktaSignOnPolicyRuleActions.js

```diff
@@ -0,0 +1,34 @@
+/*!
+ * Copyright (c) 2017-2018, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+
+/* THIS FILE IS AUTO-GENERATED - SEE CONTRIBUTOR DOCUMENTATION */
+
+var Resource = require('../resource');
+const OktaSignOnPolicyRuleSignonActions = require('./OktaSignOnPolicyRuleSignonActions');
+
+/**
+ * @class OktaSignOnPolicyRuleActions
+ * @extends Resource
+ * @property { OktaSignOnPolicyRuleSignonActions } signon
```

Comment:
nit-pick: is this case consistent? should it be signOn?

Reply:
This is auto generated code, so, all of this is based on the spec